src/fides/data/sample_project/cookie_house/src/pages/index.tsx

```diff
@@ -25,10 +31,22 @@ const IndexPage = ({
         <link rel="icon" href="/favicon.ico" />
         <link rel="stylesheet" href="https://rsms.me/inter/inter.css"></link>
       </Head>
-
+      {gtm_key && (
+        <Script id="google-tag-manager" strategy="afterInteractive">
+          {`
+            (function (w, d, s, l, i) {
+              w[l] = w[l] || []; w[l].push({
+                'gtm.start':
+                  new Date().getTime(), event: 'gtm.js'
+              }); var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+                  'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+            })(window, document, 'script', 'dataLayer', '${gtm_key}');
```

Comment:
There's some XSS risk here that I'm a bit concerned about... is there maybe something we can do to prevent the possibility of something like `gtm_key = "123'); alert('gotcha!');"` wouldn't be possible?

I suspect `next/script` might already try to prevent this, but I don't know how clever it really could be...